op-e2e/actions/user_test.go

```diff
@@ -133,7 +135,17 @@ func TestCrossLayerUser(gt *testing.T) {
 		require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status, "proposal failed")
 	}
 
-	// make the L1 side of the withdrawal tx
+	// prove our withdrawal on L1
+	alice.ActProveWithdrawal(t)
+	// mine a block with a time delta of 7 days + 1 second that has alice's
+	// proveWithdrawal tx in it.
+	miner.ActL1StartBlock(604801)(t)
```

Comment:
Is this really set to a week? You may want to change the deploy or setup parameters (`dp` / `sp`) to update this time, or assert it's set correctly, as I think the default might still be set for the real-time testing from op-e2e which allows for more instant withdrawal finalization. Testing the negative case, where it fails if someone tries to withdraw earlier than the specified finalization time, may be good to follow up with later (I think it's already captured in the testing plans, lmk if I can help specify an issue for it).

Reply:
Good catch- double checking on this now, I believe you're right about it being set to lower than a week for the sake of these tests. Agreed that it would be a good idea to add a test for the negative case, will circle back with you on that!

Reply:
RFC:

https://github.com/ethereum-optimism/optimism/blob/d200b024a0bde780c61f88c361b8b219a0ab3967/op-e2e/e2eutils/setup.go#L87

\+

https://github.com/ethereum-optimism/optimism/blob/d200b024a0bde780c61f88c361b8b219a0ab3967/op-e2e/actions/user_test.go#L138-L158

Reply:
LGTM, we can test the week finalization period boundary cases in later new tests, and cover when it's under/over that 1 week threshold. This test can just run through the core functionality with default e2e setup :+1: